app/views/search_page/search_page.erb

```diff
@@ -6,6 +6,11 @@
     locale: I18n.locale,
     defaultLocale: I18n.default_locale
   },
+  marketplace: {
+    marketplace_color1: CommonStylesHelper.marketplace_colors(@current_community)[:marketplace_color1],
```

Comment:
Good point - let's do that when we decide how to go forward with Topbar (will it be included to search page). Then we have a good time to do the naming change also helper files and topbar's props naming.